### PR TITLE
WT-11341 Chunk cache pinning documentation

### PR DESCRIPTION
**Pull request by Siddhartha8899**

### CODE REVIEW

**Comment by Siddhartha8899** [comment]
Screenshot 2023-09-07 at 4 10 36 pm

**Comment by Siddhartha8899** [comment]
Failures are unrelated.

**Review by korteland — APPROVED** [review]
LGTM!

File: src/docs/chunkcache.dox [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+are non-evictable), the pinned config is required.
+
+<b>Note:</b> It is crucial to assess the memory implications of pinning large tables. Ensure the chunkcache has enough capacity to cache the pinned tables.
+
+Example Configuration:
+@code
+chunk_cache=[enabled=true,chunk_size=512KB,capacity=20GB,pinned=("table:chunkcache01", "table:chunkcache02")]
+@endcode
+
+In the config above, the pinned config array contains the names of tables to be pinned. In this example, the tables <b>chunkcache01</b> and <b>chunkcache02</b> are specified.
```
Comment (ruby-oujo) [inline src/docs/chunkcache.dox:16]: NP: Since this is a formal document I would not use abbreviations e.g configuration.

**Review by ruby-oujo — APPROVED** [review]
LGTM, optional np comment.

File: src/docs/chunkcache.dox (hunk 1 above)
Comment (Siddhartha8899) [reply src/docs/chunkcache.dox]: Updated in [bd27d2f](https://github.com/wiredtiger/wiredtiger/pull/9613/commits/bd27d2f79f281147659b0885f36e5f06fbe9feba)

**Comment by Siddhartha8899** [comment]
evergreen merge

**Pull request merged**